src/components/nav-tabs/CommunityPage.js

```diff
@@ -1,31 +1,99 @@
 import { Button, Container, Text, Title, Group } from '@mantine/core';
 import React from 'react';
 
+import { createStyles, SimpleGrid, ThemeIcon, Grid, Col } from '@mantine/core';
+import { BuildingCommunity, MailFast, Speakerphone, BrandTwitter } from 'tabler-icons-react';
+
+const useStyles = createStyles((theme) => ({
+    wrapper: {
+      padding: `${theme.spacing.xl * 2}px ${theme.spacing.xl}px`,
+    },
+
+    title: {
+        fontFamily: `Greycliff CF, ${theme.fontFamily}`,
+        fontSize: 36,
+        fontWeight: 900,
+        lineHeight: 1.1,
+        marginBottom: theme.spacing.md,
+        color: theme.colorScheme === 'dark' ? theme.white : theme.black,
+        },
+}));
+
+const features = [
+    {
+        icon: BuildingCommunity,
+        title: <a href="https://lists.operate-first.cloud/admin/lists/community.lists.operate-first.cloud/">Community mailing list</a>,
+        description: 'for Operate First project user and contributor discussions',
+    },
+    {
+        icon: Speakerphone,
+        title: <a href="https://lists.operate-first.cloud/admin/lists/announcements.lists.operate-first.cloud/">Announcements list</a>,
+        description: 'for the latest project news',
+    },
+    {
+        icon: MailFast,
+        title: <a href="http://lists.opendev.org/cgi-bin/mailman/listinfo/openinfralabs">OpenInfra Labs mailing list</a>,
+        description:
+        'for those interested in open source cloud operations more broadly',
+    },
+    {
+        icon: BrandTwitter,
+        title: <a href="https://twitter.com/operatefirst">Twitter</a>,
+        description:
+        'You can also find us on Twitter, or view community project demos and updates on <a href="https://www.youtube.com/channel/UCe87bwqlGoBQs2RvMQZ5_sg">YouTube</a>',
```

Comment:
Seems you've nested an anchor tag into a string description
![image](https://user-images.githubusercontent.com/68972382/167898127-2580e5d1-7f1b-41b1-90c8-a13b6e602029.png)

Reply:
Also in the `Developer` and `SRE Practices` sub-pages seem to have markdown instead of using JSX so this is how it looks like

![image](https://user-images.githubusercontent.com/68972382/167898733-4b737b79-44c9-4cb8-9f01-76883982acb2.png)

I can approve as is and fix them and add some more styling to the pages with a follow-up PR. Up to you two :)

Reply:
this would be awesome. thanks

Reply:
> Seems you've nested an anchor tag into a string description
> 
> ![image](https://user-images.githubusercontent.com/68972382/167898127-2580e5d1-7f1b-41b1-90c8-a13b6e602029.png)
> 
> 

Yes I did, couldn't find a better solution/template to make a title/header clickable. Interested in a better way. Let me know.

Reply:
> Also in the `Developer` and `SRE Practices` sub-pages seem to have markdown instead of using JSX so this is how it looks like
> 
> 

> 
> 
> I can approve as is and fix them and add some more styling to the pages with a follow-up PR. Up to you two :)

Yes please.
And Open for getting help to fix this.